frontend/package.json

```diff
@@ -30,7 +30,8 @@
     "style-loader": "^0.13.0",
     "tape": "^4.5.1",
     "webpack": "^1.12.14",
-    "webpack-dev-server": "^1.14.1"
+    "webpack-dev-server": "^1.14.1",
+    "babel-plugin-transform-object-rest-spread": "^6.6.5"
```

Comment:
we have the object-rest-spread as both a dev-dependency and as a dependency.... reduce this to either one or the other?